LunaMary/src/LiteralQuiz.java

```diff
@@ -0,0 +1,27 @@
+class LiteralQuiz implements IQuestion {
```

Comment:
Same as in the other class that implements IQuestion interface. This should be renamed to something like "LiteralQuestion"